app/client-v2/src/stores/projectStore.ts

```diff
@@ -0,0 +1,132 @@
+import { getApiUrl } from "@/config";
+import {
+  COMPLETE_STATUS_TYPES,
+  type AnalysisStatus,
+  type ApiResponse,
+  type Project,
+  type ProjectSample,
+  type WorkerResponse,
+  WorkerResponseValueTypes
+} from "@/types/projectTypes";
+import { mande } from "mande";
```

Comment:
Should we be using `mande` in the other places which are doing API access?

Reply:
yup if we have another store we can if we don't need to inside a component... vueuse useFetch is available inside components however